source/core/Math.cpp

```diff
@@ -847,6 +847,14 @@ DECLARE_VIREO_PRIMITIVE2( Log2ComplexSingle, ComplexSingle, ComplexSingle, (_Par
 DECLARE_VIREO_PRIMITIVE2( ExpComplexSingle, ComplexSingle, ComplexSingle, (_Param(1) = exp(_Param(0)) ) )
 DECLARE_VIREO_PRIMITIVE3( PowComplexSingle, ComplexSingle, ComplexSingle, ComplexSingle, (_Param(2) = pow(_Param(0), _Param(1)) ) )
 DECLARE_VIREO_PRIMITIVE3( PolarComplexSingle, Single, Single, ComplexSingle, (_Param(2) = polar(_Param(0), _Param(1)) ) )
+DECLARE_VIREO_PRIMITIVE2( IsEQ0ComplexSingle, ComplexSingle, Boolean, (_Param(1) = _Param(0) == 0.0f ) )
+DECLARE_VIREO_PRIMITIVE2( IsNE0ComplexSingle, ComplexSingle, Boolean, (_Param(1) = _Param(0) != 0.0f ) )
+// The following are redundant but match LV prims and are needed so they can be polymorphic over arrays/clusters of complex
+DECLARE_VIREO_PRIMITIVE3( ComplexToPolarComplexSingle, ComplexSingle, Single, Single, _Param(1) = abs(_Param(0)); _Param(2) = arg(_Param(0)) )
+DECLARE_VIREO_PRIMITIVE4( PolarToReOrImSingle, Single, Single, Single, Single, ComplexSingle z = polar(_Param(0), _Param(1)); _Param(2) = z.real(); _Param(3) = z.imag(); )
```

Comment:
Should these oeprations like abs, polar (and any of the public APIs on Complex) handle Infinity, -Infinity, NAN, etc explicitly?